src/core/components/label/index.tsx

```diff
@@ -41,12 +47,13 @@ const Legend = ({
 	text,
 	supporting,
 	optional,
+	hideLabel,
 	cssOverrides,
 	...props
 }: LegendProps) => {
 	return (
 		<>
-			<legend css={[legend, cssOverrides]} {...props}>
+			<legend css={[legend, cssOverrides, hideLabel ? visuallyHidden : undefined]} {...props}>
```

Comment:
It's important that `cssOverrides` is the last item in this array. This ensures that the CSS properties in `cssOverrides` override all other CSS properties.